denizen_scripts/relay/repo-link/behrs_discord_toolbox.dsc

```diff
@@ -0,0 +1,22 @@
+behrs_discord_toolbox:
+  type: world
+  debug: false
+  events:
+    on discord reaction added:
+        - if <context.emoji.name> != "settings":
+            - stop
+        - if <context.author.roles[<context.message.channel.group.id>].parse[id].contains[731591651142926456]> || <context.author.roles[<context.message.channel.group.id>].parse[id].contains[679101171201474571]>:
+            - define "message:**Message Contents:** <context.message.message>"
+            - define "message:|:**UserTag (Author):** <context.message.author.as_element.replace[discorduser@].with[<&lt>discorduser<&lb>]><&rb><&gt>"
+            - define "message:|:**MessageTag:** <context.message.as_element.replace[discordmessage@].with[<&lt>discordmessage<&lb>]><&rb><&gt>"
+            - define "message:|:**ChannelTag:** <context.message.channel.as_element.replace[discordchannel@].with[<&lt>discordchannel<&lb>]><&rb><&gt>"
+            - define "message:|:**GroupTag:** <context.message.channel.group.as_element.replace[discordgroup@].with[<&lt>discordgroup<&lb>]><&rb><&gt>"
+            - if <context.message.reactions.size||0> != 0:
+                - define "message:|:**Message Reactions:**"
+                - foreach <context.message.reactions.parse[split[/].get[1]]> as:reaction:
```

Comment:
it's optimal to use the unique grabber tag `.first` over `.get[1]`
but also, if it's just grabbing what's before `/`, it's better to use the unique `before[/]` rather than splitting it and grabbing the first from the constructed list
```suggestion
                - foreach <context.message.reactions.parse[before[/]]> as:reaction:
```